### PR TITLE
[DEV APPROVED] 7643 - detailed error notifications

### PR DESCRIPTION
**Pull request by jriga**
**Feature**
Notify user that of potential corrupted file when unzipping fails

### CODE REVIEW

File: app/jobs/fca_import_job.rb [hunk 1]
```diff
@@ -66,13 +66,21 @@ def notify_email(text)
     ).deliver_later
   end
 
+  def parse_error(list)
+    if list.second.try(:success?)
+      'could not unzipped. The file could be corrupted.'
+    else
+      'technical error. Contact the dev team'
```
Comment (Tr4pSt3R) [inline app/jobs/fca_import_job.rb]: provide an email address to be contact on perhaps?

**Comment by jongilbraith** [comment]
LGTM 👍

File: app/jobs/fca_import_job.rb [hunk 2]
```diff
@@ -66,13 +66,21 @@ def notify_email(text)
     ).deliver_later
   end
 
+  def parse_error(list)
+    if list.second.try(:success?)
+      'could not unzipped. The file could be corrupted.'
```
Comment (norm) [inline app/jobs/fca_import_job.rb]: "could not *be* unzipped" or "could not unzip" :)

File: app/jobs/fca_import_job.rb (hunk 1 above)
Comment (Tr4pSt3R) [inline app/jobs/fca_import_job.rb]: Here is the Email for the dev team: development.team@moneyadviceservice.org.uk

Comment (jriga) [inline app/jobs/fca_import_job.rb]: ♻️

File: app/jobs/fca_import_job.rb (hunk 2 above)
Comment (jriga) [inline app/jobs/fca_import_job.rb]: ♻️

**Comment by Tr4pSt3R** [comment]
👍 super!

**Comment by tomas-stefano** [comment]
👍

**Pull request merged**